disdrodb/L0/retrive_reader.py

```diff
@@ -0,0 +1,96 @@
+import os

```

Comment:
We need to find a better name for this file ... or maybe add it to `L0_processing.py`

Reply:
ok I put the code in ```L0_processing.py```